src/app/components/forms/post/post.ts

```diff
@@ -353,6 +353,20 @@ export default class FormPost extends BaseForm<FormPostModel>
 		return this.defaultCommunity;
 	}
 
+	@Watch('formModel.post_to_user_profile')
+	onPostToUserProfileToggle(event: boolean) {
```

Comment:
no need to get the value here, just use this.formModel.post_to_user_profile directly in the function body. It'll be up to date and this way the typing is synced properly.